### ISSUE TITLE
 Destructuring expressions can only have identifier references

### ISSUE THREAD

**Issue by izaa** [VeliovGroup/Meteor-Files#663]
I'm having an issue on Client side
- if opening a Meteor project that uses Meteor-Files package, Microsoft Edge wont open the site. Gives an error

>  SCRIPT1072: Destructuring expressions can only have identifier references 
> ostrio_files.js (131,14)

I haven't tested yet, if the (perhaps reserved) public-keyword below causes the error?

```
class FilesCollection extends FilesCollectionCore {
  constructor(config) {
    super();

    if (config) {
      ({
        ddp: this.ddp,
        debug: this.debug,
        schema: this.schema,
        public: this.public,
        chunkSize: this.chunkSize,
        collection: this.collection,
        downloadRoute: this.downloadRoute,
        disableUpload: this.disableUpload,
        namingFunction: this.namingFunction,
        collectionName: this.collectionName,
        onBeforeUpload: this.onBeforeUpload,
        allowClientCode: this.allowClientCode,
        onbeforeunloadMessage: this.onbeforeunloadMessage
      } = config);
    }
```

Meteor-Files version: ostrio:files@1.10.2
Meteor 1.8
Microsoft Edge 40.15063.674.0 on Windows 10 
Client issue

**Comment by dr-dimitru** [comment]
Send a PR, we will merge quickly.

**Comment by izaa** [comment]
I have tested this in my environment, havent had time to create PR yet but 

in `client.js `

changed
```

({
        ddp: this.ddp,
        debug: this.debug,
        schema: this.schema,
        public: this.public,
        chunkSize: this.chunkSize,
        collection: this.collection,
        downloadRoute: this.downloadRoute,
        disableUpload: this.disableUpload,
        namingFunction: this.namingFunction,
        collectionName: this.collectionName,
        onBeforeUpload: this.onBeforeUpload,
        allowClientCode: this.allowClientCode,
        onbeforeunloadMessage: this.onbeforeunloadMessage,
      } = config);

```
to 

```
      Object.keys(config).forEach(c => {
        this[c] = config[c];
      });
```


seems to work

**Comment by dr-dimitru** [comment]
@izaa thank you for reporting about this issue.
It is solved and released as [`v1.11.1`](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.11.1)

Feel free to reopen it in case if the issue is still persists on your end.
